src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs

```diff
@@ -16,6 +16,8 @@ namespace OrchardCore.Content.Controllers
     [Authorize(AuthenticationSchemes = "Api"), IgnoreAntiforgeryToken, AllowAnonymous]
     public class ApiController : Controller
     {
+        private static readonly JsonMergeSettings UpdateJsonMergeSettings = new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace };
```

Comment:
Use `_updateJsonMergeSettings` because this is the convention we 're using for `private` fields

Reply:
not the convention we seem to be using so much for static privates.